ledger/participant-integration-api/src/main/scala/platform/store/dao/events/TransactionsReader.scala

```diff
@@ -250,20 +262,28 @@ private[dao] final class TransactionsReader(
       .flatMap(rawEvents =>
         Timed.value(
           timer = dbMetrics.lookupTransactionTreeById.translationTimer,
-          value = Future.traverse(rawEvents)(deserializeEntry(verbose = true)),
+          value = Future.traverse(rawEvents)(
+            deserializeEntry(
+              EventProjectionProperties(
+                verbose = true,
+                requestingParties.map(_.toString -> Set.empty[Identifier]).toMap,
+              )
+            )
+          ),
         )
       )
       .map(TransactionConversions.toGetTransactionResponse)
 
   override def getActiveContracts(
       activeAt: Offset,
       filter: FilterRelation,
-      verbose: Boolean,
+      eventProjectionProperties: EventProjectionProperties,
   )(implicit loggingContext: LoggingContext): Source[GetActiveContractsResponse, NotUsed] = {
     val contextualizedErrorLogger = new DamlContextualizedErrorLogger(logger, loggingContext, None)
     val span =
       Telemetry.Transactions.createSpan(activeAt)(qualifiedNameOfCurrentFunc)
-    logger.debug(s"getActiveContracts($activeAt, $filter, $verbose)")
+
+    logger.debug(s"getActiveContracts($activeAt, $filter, $eventProjectionProperties)")
```

Comment:
I am not sure what to do here with the logging... what is important then later to be shown?